meetings/2024-06-26.md

```diff
@@ -0,0 +1,89 @@
+# TC39-TG5-4 2024-06-26
+
+**Remote attendees:**
+| Name              | Abbreviation | Organization     |
+| ----------------- | ------------ | ---------------- |
+| Mikhail Barash    | MBH          | UiB              |
+| Yulia Startsev    | YSV          | Mozilla          |
+| Michael Ficarra   | MF           | F5               |
+| Jesse Alama       | JMN          | Igalia           |
+| Michael Dyck      | MD           |                  |
+| Juho Vepsäläinen  | JV           | Aalto University |
+| Michael Coblenz   | MC           | UCSD             |
+| Eemeli Aro        | EAO          | Mozilla          |
+
+## Meeting:
+
+### Subject: Editors' work on refining the formalism of the spec
+
+MF: (presentation)
+  * [(§5.2.5)](https://262.ecma-international.org/#sec-mathematical-operations) numeric types
+  * [(§6.2.2)](https://262.ecma-international.org/#sec-list-and-record-specification-type) proliferation of records (reducing core concepts)
+  * [(§6.2.1)](https://262.ecma-international.org/#sec-enum-specification-type) spec enums
+  * [(§5.2.7)](https://262.ecma-international.org/#sec-identity) identity
+  * [(§6.2.4)](https://262.ecma-international.org/#sec-completion-record-specification-type) explicit completion record handling
+  * [(§6.2.8)](https://262.ecma-international.org/#sec-abstract-closure) abstract closures
+  * AO type signatures
+  * explicit, principled integration points
+	  - host hooks
+	  - (Annex B) step replacements for normative optional / legacy
+	  - [(#2952)](https://github.com/tc39/ecma262/pull/2952) feature flags for joint requirements
+
+YSV: [executable specification]
+
+MF: [editors' use of the KAIST's tool]
+
+JMN: I'd love to see those math functions nailed down a bit more, but maybe that horse has left the barn?
+
+MF: [we review manually]
+
+MF: working on math functions (not as an editor). Fully define math sqr, other math functions,
+plan on introducing constraints on them, bounds on return values, monotonicity
+
+MC (in chat): It seems like really doing this would require translating the whole thing to a formal specification (e.g. in Coq).
+
+MD (in chat): Another barrier to machine execution is that it's difficult to extract the characteristics
+of the intrinsic objects and the relations between them.
+
+MF (in chat): It sounds like part of the problem is that the machine-readable specifications approaches are too hard for humans to read.
+I wonder if research could make machine-interpretable specs easier for humans to read.
```

Comment:
I think this was someone else? I do remember briefly mentioning SpecTec at this point, but not this.

Reply:
```suggestion
[message in chat]: It sounds like part of the problem is that the machine-readable specifications approaches are too hard for humans to read. I wonder if research could make machine-interpretable specs easier for humans to read.
```